src/stackedBars.ts

```diff
@@ -48,14 +52,55 @@ export function renderStackedBars(pareto: Pareto, settings: Settings) {
         .attr("shape-rendering", "crispEdges")
         .attr("stroke", (d) => (d.isMarked ? "#000" : "none"))
         .attr("stroke-width", (d) => (d.isMarked ? settings.style.selectionBox.strokeWidth : "0"));
-
+        
+    
+        
+     const range =  svgBoundingClientRect.width - resources.PADDINGRIGHT 
+     const percentageLabelPadding = svgBoundingClientRect.width /4;
+     let ppp = categoryAxisBandwidth.bandwidth()/4;

```

Comment:
Can we have a more descriptive variable name?

Reply:
I changed them